src/main/java/org/opensearch/security/securityconf/SecurityRoles.java

```diff
@@ -55,6 +55,8 @@ boolean hasExplicitIndexPermission(
 
     Set<String> getRoleNames();
 
+    void addRole(Role role);
```

Comment:
This can be removed assuming `Role.java` is dropped.

Reply:
This is required since getSecurityRoles method calls addRole method internally